lxd/auth/drivers/openfga.go

```diff
@@ -134,11 +134,6 @@ func (e *embeddedOpenFGA) CheckPermission(ctx context.Context, entityURL *api.UR
 		return fmt.Errorf("Failed to parse entity URL: %w", err)
 	}
 
-	err = auth.ValidateEntitlement(entityType, entitlement)
```

Comment:
do we have a test for trying to assign an invalid entitlement? (to check openfga does indeed block this)

Reply:
Do you mean a check for say:
```
! lxc auth group permission add instance_snapshot c1-snap can_view || false # Cannot be assigned via API
```

Or a check where we manipulate the `auth_groups_permissions` table to insert a permission that is invalid from the perspective of OpenFGA?